README.md

```diff
@@ -9,6 +9,27 @@ A Visual Studio Code extension that allows you to use the browser's Elements and
 
 ![Microsoft Edge Tools - Demo](basic_usage.gif)
 
+* [Supported Features](#supported-features)
```

Comment:
Personally, I'd think it'd look nice with a header for the ToC like [this](https://github.com/ekalinin/github-markdown-toc#table-of-contents):

![image](https://user-images.githubusercontent.com/14304598/106652883-f3f77c00-654a-11eb-9123-df68eec26d01.png)

Reply:
Added it.